assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -143,15 +143,15 @@ che.workspace.activity_cleanup_scheduler_initial_delay_s=60
 
 
 # Delay before first workspace idleness check job started to avoid
-# mass suspend if ws master was unavailable for period close to
+# mass suspend if {prod-short} server was unavailable for period close to
```

Comment:
can we please put `{prod-short}` in the list on the top of file so it's clear to what it is translated? https://github.com/eclipse-che/che-server/blob/main/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L13